load.php

```diff
@@ -70,7 +70,6 @@ public static function initExtension() {
 
 		// Because of MW 1.19 we need to register message files here
 		$GLOBALS['wgMessagesDirs']['SemanticMediaWiki'] = $GLOBALS['smwgIP'] . 'i18n';
-		$GLOBALS['wgExtensionMessagesFiles']['SemanticMediaWiki'] = $GLOBALS['smwgIP'] . 'languages/SMW_Messages.php';
```

Comment:
```
There was 1 failure:
1) SMW\Tests\SetupTest::testRegisterMessageFiles with data set #0 ('SemanticMediaWiki', array(array('')))
Failed asserting that false is true.
```

This line is checked in [0], so removing `SemanticMediaWiki` from [0] should make the test pass.

[0] https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/tests/phpunit/includes/SetupTest.php#L279